File: WebScrapper.AngularJs/WebScrapper.AngularJs/app/models/Action.js
var app;
(function (app) {
    var models;
    (function (models) {
        var Action = (function () {
            function Action(id, url, httpMethod, parameters) {
                this.id = id;
                this.url = url;
                this.httpMethod = httpMethod;
                this.parameters = parameters;
            }
            return Action;
        }());
        models.Action = Action;
    })(models = app.models || (app.models = {}));
})(app || (app = {}));

File: WebScrapper.Api/WebScrapper.core/Layer/Domain/Actions/Action.cs
using System;
using System.Collections.Generic;
using WebScrapper.Core.Helpers.Application;
using WebScrapper.Core.Helpers.Domain;
using WebScrapper.Core.Layer.Application.ResponseActions;
using WebScrapper.Core.Layer.Domain.ResponseActions;

namespace WebScrapper.Core.Layer.Domain.Actions
{
    public class Action : IDomainEntity
    {
        public Guid Id { get; set; }

        public String Url { get; set; }

        public String HttpMethod { get; set; }

        public IList<Parameter> Parameters { get; set; }

        public ResponseAction ResponseAction { get; set; }
    }
}